pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/dictionary/StringOnHeapMutableDictionary.java

```diff
@@ -157,7 +157,7 @@ public String getStringValue(int dictId) {
 
   @Override
   public byte[] getBytesValue(int dictId) {
-    return BytesUtils.toBytes(getStringValue(dictId));
+    return getStringValue(dictId).getBytes(StandardCharsets.UTF_8);
```

Comment:
(nit) `import static java.nio.charset.StandardCharsets.UTF_8;` to be consistent with other classes

Reply:
Done